web/src/app/shared/converters/loi-converter/loi-data-converter.spec.ts

```diff
@@ -218,16 +213,16 @@ describe('loiToJS_Error', () => {
       `got unexpected error in geometry conversion ${geoPointWithError}`
     );
   }
-  const geoPoint = (geoPointWithError as unknown) as Geometry;
+  const geoPoint = (geoPointWithError as unknown) as GeoPoint;
 
   const testData: {
     expectation: string;
     loi: LocationOfInterest;
     wantErrorMessage: string;
   }[] = [
     {
-      expectation: 'Cannot convert generic LOI',
-      loi: new GenericLocationOfInterest(
+      expectation: 'Cannot convert PointOfInterest',
```

Comment:
Not sure if we should keep this test case, maybe add a TODO to remind us deleting it after all the migration is done?

Reply:
I think maybe we can simplify this once the migration is done, but it's still useful to test at least one error case.